sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -60,12 +60,28 @@ static const bool UseCopyEngineForD2DCopy = [] {
   return (CopyEngineForD2DCopy && (std::stoi(CopyEngineForD2DCopy) != 0));
 }();
 
+// TODO: Add support for non-zero values of SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE
```

Comment:
But you already support 1-digit non-zero values below. Comment outdated? Also why need this check for exactly 1 symbol?

Reply:
Removed the unnecessary check and the comment. Thanks